integration_test/gce/gce_testing.go

```diff
@@ -1285,7 +1303,7 @@ sudo zypper --non-interactive refresh test-vendor`
 	installCmd := `set -ex
 
 ` + repoSetupCmd + `
-sudo zypper --non-interactive install --force-resolution --capability 'python>=3.6'
```

Comment:
I think this was also a recent change. Do you need to merge in master?

Reply:
bad merge